src/net/skbuff/skb_data.c

```diff
@@ -113,10 +108,11 @@ struct sk_buff_data * skb_data_alloc(size_t size) {
 	skb_data->alloc_type = alloc_type;
 	skb_data->links = 1;
 
-	return skb_data;
+	return (void *)skb_data;
```

Comment:
`(void *)s...`->`(void *) s...`

Reply:
In my mind
`(cast type)var` looks better than '(cast type) var'